Makefile

```diff
@@ -7,6 +7,8 @@ SOURCES = $(BASENAME).txt
 TARGETS = epub html pdf readme
 DOCBOOK_DIST = xsl/xsl-ns-stylesheets
 
+RUDDER_VERSION = 3.0
```

Comment:
When you add a version specific information into some file, do not forget that it needs to be updated on each version change, which is hard/boring/easily forgotten ...
So you should preferably find a way to guess it automatically, or if you can't, add the information into a manual procedure.
For this one, I've seen with @VinceMacBuche and we've added it to the release procedure.